lib/cisco_node_utils/cmd_ref/bgp.yaml

```diff
@@ -1,109 +1,197 @@
 # bgp.yaml
 ---
 _template:
-  config_get: "show running bgp all"
-  config_get_token: '/^router bgp <asnum>$/'
-  config_get_token_append:
-    - '/^vrf <vrf>$/'
-  config_set: "router bgp <asnum>"
-  config_set_append:
-    - "vrf <vrf>"
+  grpc:
+    config_get: "show running-config router bgp"
+    config_get_token: '/^router bgp <asnum>$/'
+    config_get_token_append:
+      - '/^vrf <vrf>$/'
+    config_set: "router bgp <asnum>"
+    # Note the space before 'vrf' - seems like grpc
+    # is currently indentation sensitive.
+    config_set_append:
+      - " vrf <vrf>"
+  nxapi:
+    config_get: "show running bgp all"
+    config_get_token: '/^router bgp <asnum>$/'
+    config_get_token_append:
+      - '/^vrf <vrf>$/'
+    config_set: "router bgp <asnum>"
+    config_set_append:
+      - "vrf <vrf>"
 
 address_family:
   config_set_append: '<state> address-family <afi> <safi>'
 
 bestpath_always_compare_med:
-  config_get_token_append: '/^bestpath always-compare-med$/'
-  config_set_append: '<state> bestpath always-compare-med'
-  default_value: false
+  grpc:
+    config_get_token_append: '/^bgp bestpath med always$/'
+    config_set_append: ' <state> bgp bestpath med always'
+    default_value: false
+  nxapi:
+    config_get_token_append: '/^bestpath always-compare-med$/'
+    config_set_append: '<state> bestpath always-compare-med'
+    default_value: false
```

Comment:
ditto - `default_value` is `false` for both so it doesn't need to be repeated. Same suggestion throughout below.